src/Catty/Parser&Serialializer/New Parser & Serializer/XMLHandler/Bricks/LookBricks/SetBackgroundByIndexBrick+CBXMLHandler.swift

```diff
@@ -0,0 +1,42 @@
+/**
+ *  Copyright (C) 2010-2021 The Catrobat Team
+ *  (http://developer.catrobat.org/credits)
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  An additional term exception under section 7 of the GNU Affero
+ *  General Public License, version 3, is available at
+ *  (http://developer.catrobat.org/license_additional_term)
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see http://www.gnu.org/licenses/.
+ */
+
+extension SetBackgroundByIndexBrick: CBXMLNodeProtocol {
+
+    static func parse(from xmlElement: GDataXMLElement, with context: CBXMLParserContext) -> Self {
+        CBXMLParserHelper.validate(xmlElement, forNumberOfChildNodes: 1, andFormulaListWithTotalNumberOfFormulas: 1)
+        let formula = CBXMLParserHelper.formula(in: xmlElement, forCategoryName: "LOOK_INDEX", with: context)
```

Comment:
In the meantime Catroid has renamed `LOOK_INDEX` to `BACKGROUND_INDEX`. Please rename as well.

Reply:
The name stays `BACKGROUND_INDEX`, however to support older projects, please also allow the parsing (not serializing) of `LOOK_INDEX`. So both should be possible (see CATROID-977). Please also add both version to _ValidProjectAllBricks0992.xml_.